docs/REST/TEC/V1/creating-endpoints.md

```diff
@@ -57,7 +58,14 @@ class MyEntity extends Post_Entity_Endpoint implements RUD_Endpoint {
      * Returns the base path for the endpoint.
      */
     public function get_base_path(): string {
-        return '/my-entities/%s';
+        return '/my-entities/{id}';
+    }
```

Comment:
Same as the other PR: Since this is in the context of WordPress, these examples should follow WP coding standards, e.g. tabs instead of spaces.